src/components/Request/FormFields/PositionSensitivity.tsx

```diff
@@ -22,6 +22,7 @@ const PositionSensitivity = ({ name, form }: FormField) => {
         control={form.control}
         render={({ field }) => (
           <Combobox
+            required
```

Comment:
Leverage RHF validation like the other controls
```suggestion
        control={form.control}
        rules={{
          required: "Position Sensitivity is required",
        }}
        render={({ field }) => (
          <Combobox
```